blueoil/blueoil_convert.py

```diff
@@ -46,10 +46,11 @@ def get_export_directory(experiment_id, restore_path):
     return export_dir
 
 
-def create_output_directory(output_root_dir):
+def create_output_directory(output_root_dir, output_template_dir=None):
     """Create output directory from template."""
 
-    template_dir = "/home/blueoil/output_template"
+    # TODO: Remove the hard-coded path.
+    template_dir = "/home/blueoil/output_template" if not output_template_dir else output_template_dir
```

Comment:
Can you use TEMPLATE_DIR instead of  `/home/blueoil/output_template` ?

Reply:
@ruimashita

What `TEMPLATE_DIR` you are talking about is it?
https://github.com/blue-oil/blueoil/blob/741a5bd93e2210c64167dddc22078ee2c50d72dc/blueoil/vars.py#L19-L20

This is the template path of the blueoil/lmnet config file.
It is the template of the conversion result (run.py, usb_camera_demo.py, etc...) that we must specify with `output_template_dir`.

Reply:
@odoku 
Oh, I mistaken 😭 `TEMPLATE_DIR` is not `/home/blueoil/output_template` .
In order to remove hard-code, Could you create new constant like `OUTPUT_TEMPLATE_DIR` ?

Reply:
@ruimashita I fixed it. Please check again! https://github.com/blue-oil/blueoil/pull/105/commits/cdf77ce36eff53a31b6961751a183a8a748b7502

Reply:
@odoku Thanks